### DISCUSSION TITLE
Blitz replace Wordpress ???

### DISCUSSION THREAD

**Discussion by IRediTOTO** [blitz-js/blitz#3003] [Q&A] [answered]
Hi, I just found this, it look so amazing.
Look like receipt feature can generate a complete theme same as WP theme. 
I just wonder can Blitz will be a WP alternative option ?

**Comment by beerose** [comment, marked as answer]
I'm glad you like Blitz!

Regarding Blitz vs WordPress, there are a few things to consider. WordPress is a content management system that allows you to build and host a website without requiring you to write code (thanks to the plugin system that lets you add various customization). While Blitz is a React framework. The most significant difference between React frameworks (Blitz, Next, Gatsby) and WordPress is that they have wholly separated the website’s content management system from the front end (the interface) of a website. In order to display content, you have to either fetch data from the backend (which is super easy with Blitz) or use a headless CMS solution.

I believe that there are use cases that can be covered by both Blitz and WordPress, but in some cases, one of these tools might work better. If you are a blogger and only want a few pages on your site, and you don't want to write code, I'd say using WordPress may be a good idea. However, if you want to build a highly customizable website, have a fully-fledged backend and an easy access to a database, then Blitz can be something you're looking for.

**Reply by IRediTOTO** [reply]
But if someone make hole backend / frontend system like wp base on blitz, write it down to Blitz receipt feature. Just run install => connect db and bump, we got a full website. Still fully customizable. I know WP is so BIG CMS, but i am waiting for something replace it :D

**Reply by beerose** [reply]
I know that some people are working on CMS-like apps on Blitz, so maybe we'll have a nice setup as a recipe in the future!

**Reply by rockiger** [reply]
> I know that some people are working on CMS-like apps on Blitz, so maybe we'll have a nice setup as a recipe in the future!

Are they building in the open, or are these private projects? If open, could you name the project.

**Reply by beerose** [reply]
I'm not sure, @rockiger. I saw some conversations in Blitz discord (https://discord.blitzjs.com/) about CMS apps on Blitz, but I don't know any details.

**Reply by prisis** [reply]
> > I know that some people are working on CMS-like apps on Blitz, so maybe we'll have a nice setup as a recipe in the future!
> 
> Are they building in the open, or are these private projects? If open, could you name the project.

I'm building one, if I finish the base features, it will be available on https://github.com/visulima

Her is a small preview

https://media.discordapp.net/attachments/802989186495152149/910663505676492800/cms_preview.png

**Reply by beerose** [reply]
Thank you for sharing @prisis!

**Reply by IRediTOTO** [reply]
@prisis hope you can make a 2nd WP :) and price not too high

**Reply by prisis** [reply]
Was thinking to have a free version if self hosted and paid hosting, something like ghost


Checking all open source cms, to get a filling what feature are needed.

**Reply by IRediTOTO** [reply]
Why dont you follow Wp? They have wp.com which is paid service and wp.org for free open source. I think that can make you growth so fast

**Reply by IRediTOTO** [reply]
Btw still wait for some years to see your project apear :/
